src/Fable.Compiler/Fable.Compiler.fsproj

```diff
@@ -0,0 +1,56 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+        <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+        <RootNamespace>Fable.Compiler</RootNamespace>
+        <Version>0.1.0</Version>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+        <!-- Generate PDB files in release mode -->
+        <DebugType>portable</DebugType>
```

Comment:
@nojaf Perhaps we can keep the PDBs embedded here too (seems like that's the default, at least on .NET 8.0).

Reply:
Where are you getting that? I believe the default is `portable`.
I'm fine with embedded, all I really want to do is be able to step through the `Fable.Compiler` and `FCS` code when debugging.

Reply:
@nojaf I think you're right, the default seems to be `portable`.
Not sure how FCS sets it to `embedded`, but that's not important.

I guess we don't need to set it to `portable` here then, if `portable` is the default.